src/redux/features/user/userSelectors.ts

```diff
@@ -0,0 +1,3 @@
+import { RootState } from 'src/redux/rootReducer';
+
+export const selectUser = (state: RootState) => state.user;
```

Comment:
can we call this something other than selectUser? We have a concept of a user in the app and it is not this. Call it maybe selectUserAnalytics? and state.userAnalytics?